srcpkgs/guile-wisp/template

```diff
@@ -1,16 +1,20 @@
 # Template file for 'guile-wisp'
 pkgname=guile-wisp
-version=1.0.2
+version=1.0.3
 revision=1
-wrksrc="wisp-${version}"
+wrksrc="wisp-v${version}"
 build_style=gnu-configure
 configure_args="--datarootdir=/usr/share"
-hostmakedepends="guile pkg-config python3"
-makedepends="guile-devel"
-depends="guile>=2.0"
```

Comment:
Why remove depends completely?

Reply:
I guess you've already [guessed why](https://github.com/void-linux/void-packages/pull/27320#pullrequestreview-558849450) :)